src/parser/jobs/rpr/modules/DefensiveCooldowns.tsx

```diff
@@ -0,0 +1,10 @@
+import {Defensives} from 'parser/core/modules/Defensives'
+
+export class DefensiveCooldowns extends Defensives {
+	protected override trackedDefensives = [
+		this.data.actions.ARCANE_CREST,
+		this.data.actions.FEINT,
+		this.data.actions.SECOND_WIND,
+		this.data.actions.BLOODBATH,
+	]
+}
```

Comment:
Nit; Why did you remove the Info part of Defensives? It's not a bad idea to explain at a minimum how Arcane Crest and Feint can support the party like it was doing before.